src/DataLayer/converters/toAfmResultSpec.ts

```diff
@@ -21,6 +23,10 @@ function convertAttribute(
 
 function convertAFM(visualizationObject: VisualizationObject.IVisualizationObjectContent): AFM.IAfm {
     const attributes: AFM.IAttribute[] = getAttributes(visualizationObject.buckets).map(convertAttribute);
+    const { controls: { tooltipText = "" } = {} } = parsePropertyItem(visualizationObject.properties || "");
+    if (tooltipText) {
+        attributes.push(buildTooltipBucketItem(tooltipText));
+    }
```

Comment:
It could be
```
function convertAFM(visualizationObject: VisualizationObject.IVisualizationObjectContent): AFM.IAfm {
    const textualAttributes: AFM.IAttribute[] = getAttributes(visualizationObject.buckets).map(
        convertAttribute,
    );
    const geoAttribute = getGeoAttributeForTooltip(visualizationObject);
    const attributes = geoAttribute ? [...textualAttributes, geoAttribute] : textualAttributes;

    const attrProp = attributes.length ? { attributes } : {};

    ...
}

function getGeoAttributeForTooltip(
    visualizationObject: VisualizationObject.IVisualizationObjectContent,
): AFM.IAttribute | null {
    const properties: IVisualizationProperties = parsePropertyItem(visualizationObject.properties || "");
    const tooltipText: string = get(properties, "controls.tooltipText", "");
    if (tooltipText) {
        return buildTooltipBucketItem(tooltipText);
    }
    return null;
}

```